admin_manual/configuration/server/occ_command.rst

```diff
@@ -867,15 +878,25 @@ The syntax is::
 
   group:list-members [options] <group>
 
-This example lists members of the Finance group:: 
+Options:
+
+::
+  
+  --output[=OUTPUT]  Output format (plain, json or json_pretty, default is plain) [default: "plain"]
+
+This example lists members of the Finance group
+
+:: 
  
  sudo -u www-data php occ group:list-members Finance
   - aaron: Aaron Smith
   - julie: Julie Jones
 
-The output can be formatted in JSON with the output option ``json`` or ``json_pretty``::
+This example lists members of the Finance group formatted in ``json_pretty``
```

Comment:
s/This example lists groups containing the string finance formatted in json_pretty/This example lists groups containing the string "finance" formatted with ``json_pretty``/g